src/catalog/database_catalog.cpp

```diff
@@ -1,18 +1,19 @@
-#include "catalog/database_catalog.h"
-
 #include <memory>
 #include <string>
 #include <utility>
 #include <vector>
 
 #include "catalog/catalog_defs.h"
+#include "catalog/database_catalog.h"
```

Comment:
Shouldn't this be at the top?

Reply:
I think this is a `clang-format` version discrepancy. I don't think the version of `clang-format` that we're using in our toolchain enforces that ordering, while the newer one bundled with CLion does which is why you see that ordering in quite a few files. I suspect we'll eventually standardize on it when we bump the toolchain next, but for now it's probably too difficult to enforce outside of manual code review.